File: DSA/prb_3/reverse_string.py
'''
Given a string `s`, return the reversed string.

Example:
Input: "hello"
Output: "olleh"

'''

# Approch- slicing
def  reverse_string(str):
    return str[::-1]

result = reverse_string("hello")
print(result)

#Approch-2
def reverse_str(str1):
    rs_string = "";

    for ch in str1:
        rs_string = ch + rs_string

    return rs_string

result = reverse_str("HELLO")
print(result) 

File: DSA/prb_3/reverse_String.java
package DSA.prb_3;

public class reverse_String {
    
    public static String simpleReverse(String s){
        return new StringBuilder(s).reverse().toString();
    }

    public static void main(String[] args) {
        String s = "hello";
        String rs = simpleReverse(s);
        System.out.println("Reversed String : "+ rs);
    }
}
